eng/common/TestResources/New-TestResources.ps1

```diff
@@ -119,6 +119,8 @@ $root = [System.IO.Path]::Combine($repositoryRoot, "sdk", $ServiceDirectory) | R
 $templateFileName = 'test-resources.json'
 $templateFiles = @()
 $environmentVariables = @{}
+# Azure SDK Developer Playground
+$defaultSubscription = "faa080af-c1d8-40ad-9cce-e1a450ca5b57"
```

Comment:
Will this work for partner teams?

Reply:
I thought we were giving them access. If not they can log in using `Connect-AzAccount` on their own.

Reply:
@mitchdenny @weshaggard  do you happen to know if partner teams will have access to Azure SDK Dev Playground subscription?

Reply:
Based on the current permissions set I believe that its just ADP. Generally folks have contributor access for our org and then a few folks have ownership.

Reply:
I'm fine with defaulting to this subscription but can we please test the failure case for someone that doesn't have access to this default sub (or another test sub)? I want to make sure it is pretty clear what is going wrong and not failing in some very hard to diagnose way.

Reply:
Yeah, it fails with pretty clear access denied message.